wrappers/node/test/helpers/utils.ts

```diff
@@ -55,3 +59,17 @@ export const shouldThrow = (fn: () => any): Promise<vcx.VCXInternalError> => new
 })
 
 export const sleep = (timeout: number) => new Promise((resolve, reject) => setTimeout(resolve, timeout))
+
+
+let garbageCollectionBeforeExitIsScheduled = false
+
+// For some reason, The Rust library segfaults if global.gc() is not called explicitly.
```

Comment:
Perhaps just note that this only happens on exit and the trick only works for Mocha

Reply:
Changed to a better comment.